lib/logstash/inputs/meetup.rb

```diff
@@ -44,10 +44,11 @@ def register
 	addon = "venue_id=#{ @venueid }"
     else
     # None Selected, raise an error
+    raise "Configuration error! -  Must have one of `urlname`, `venue_id`, or `group_id` defined"
```

Comment:
indentation?

Reply:
@colinsurprenant - good eye..fixed.  The whole file had funky formatting, so I just ran my formatter on it. https://github.com/logstash-plugins/logstash-input-meetup/pull/12/commits/c8324c39247780808e833f3ca9018a3cf4c1d7a9 => format changes only.